islandora_plupload.module

```diff
@@ -107,9 +125,14 @@ function islandora_plupload_element_validate($element, &$form_state) {
   elseif (!is_numeric($element_value)) {
     $element_value = 0;
     if ($element['#upload_required']) {
-      form_error($element, t('@name field is required.', array('@name' => $element['#title'])));
+      $messages[] = t('@name field is required.', array('@name' => $element['#title']));
     }
   }
+  // Drupal will only print one error per element.
+  $message = implode('<br/>', $messages);
+  if ($message) {
+    form_error($element, filter_xss($message, array('br')));
```

Comment:
We don't actually get much by attaching all the messages to `plupload` elements: they don't do the highlighting thing. Would it be more convenient to do a single `form_error()` (one of either the ["There were validation issues"](https://github.com/discoverygarden/islandora_plupload/pull/16/files#diff-d1c2ac226ae0e52fadd210d666dc1883R110) or ["It's required"](https://github.com/discoverygarden/islandora_plupload/pull/16/files#diff-d1c2ac226ae0e52fadd210d666dc1883R128) bits), with [the rest](https://github.com/discoverygarden/islandora_plupload/pull/16/files#diff-d1c2ac226ae0e52fadd210d666dc1883R106) just output via `drupal_set_message()` with the `error` status?